<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppConfigView</class>
 <widget class="QWidget" name="AppConfigView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="GroupBox" name="gpb_Config">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>800</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="lbl_PathB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>161</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>DataPath_Backup</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_SaveConfig">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>420</y>
      <width>130</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Config</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Path">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>161</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>DataPath</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="led_Path">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="led_PathB">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="led_IP">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_IP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>161</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Server.IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="led_Port">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Port">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>161</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Server.Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="led_LocalIP">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_LocalIP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>161</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Local IP</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupBox</class>
   <extends>QGroupBox</extends>
   <header>CustomerWidget/GroupBox/groupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
